ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -173,10 +192,51 @@
   [context]
   (bulk-update/cleanup-old-bulk-update-status context))
 
+(defn- process-subscriptions
+  "Process each subscription in subscriptions."
+  [context subscriptions time-constraint]
+  (for [subscription subscriptions
+       :let [email-address (:email-address subscription)
+             coll-id (:collection-concept-id subscription)
+             query-params (create-query-params (:metadata subscription))
+             params1 (merge {"created-at" time-constraint}
```

Comment:
This is a little confusing, why are there two separate granule reference searches?  Maybe add a comment here to explain the weirdness.

Reply:
This is because our search queries are all "Anded", no "Or" option available. I need the granule that are created or updated during a period...  We have a json query option for all the or and and but only working for collections.